files/image_config/warmboot-finalizer/finalize-warmboot.sh

```diff
@@ -108,14 +108,10 @@ function restore_counters_folder()
 {
     debug "Restoring counters folder after warmboot..."
 
-    modules=("portstat-0" "dropstat" "pfcstat-0" "queuestat-0" "intfstat-0")
-    for module in ${modules[@]}
-    do
-        statfile="/host/counters/$module"
-        if [[ -d $statfile ]]; then
-            mv $statfile /tmp/
-        fi
-    done
+    cache_counters_folder="/host/counters"
+    if [[ -d $cache_counters_folder ]]; then
+        mv $cache_counters_folder /tmp/cache
```

Comment:
Do we need to remove `/tmp/cache` first? Otherwise, we may see below error
```
~$ sudo mv /host/counters /tmp/cache
mv: inter-device move failed: '/host/counters' to '/tmp/cache/counters'; unable to remove target: Directory not empty
```

Reply:
@bingwang-ms This script may run only after swss has started, thus the /tmp/cache is already cleared if it existed for some reason in swss.sh.